app/build.gradle.kts

```diff
@@ -0,0 +1,147 @@
+import com.android.build.gradle.internal.tasks.factory.dependsOn
+import com.github.spotbugs.snom.SpotBugsTask
+
+plugins {
+    id("com.android.application")
+    id("com.github.spotbugs")
+}
+
+spotbugs {
+    ignoreFailures.set(false)
+    setEffort("max")
+    excludeFilter.set(file("./config/spotbugs/exclude.xml"))
+    reportsDir.set(file("$buildDir/reports/spotbugs/"))
+}
+
+android {
+    namespace = "protect.card_locker"
+    compileSdk = 33
+
+    defaultConfig {
+        applicationId = "me.hackerchick.catima"
+        minSdk = 21
+        targetSdk = 33
+        versionCode = 131
+        versionName = "2.26.0"
+
+        vectorDrawables.useSupportLibrary = true
+        multiDexEnabled = true
+
+        resourceConfigurations += listOf("ar", "bg", "bn", "bn-rIN", "bs", "cs", "da", "de", "el-rGR", "en", "eo", "es", "es-rAR", "fi", "fr", "he-rIL", "hi", "hr", "hu", "in-rID", "is", "it", "ja", "ko", "lt", "lv", "nb-rNO", "nl", "oc", "pl", "pt", "ro-rRO", "ru", "sk", "sl", "sv", "tr", "uk", "zh-rTW", "zh-rCN")
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = true
+            proguardFiles(
+                getDefaultProguardFile("proguard-android.txt"),
+                "proguard-rules.pro"
+            )
+            resValue("string", "app_name", "Catima")
+        }
+        debug {
+            applicationIdSuffix = ".debug"
+            resValue("string", "app_name", "Catima Debug")
+        }
+    }
+
+    buildFeatures {
+        buildConfig = true
```

Comment:
Note to self: this seems to be moved from `gradle.properties` to here, this is not new.